README.md

```diff
@@ -6,52 +6,66 @@ Try the current version at: [https://cs121-team-panda.github.io/coconut-interpre
 
-The interpreter consists of a simple, elegant webpage that enables users to easily enter and execute Coconut code.
+**1 - Compilation Settings:** Clicking the settings gear will display a settings drawer. From within the drawer, the user can select which version of Python the Coconut code will compile to. By default, the latest version of Python is selected. 
 
-![ui](https://user-images.githubusercontent.com/35832643/37873776-fcecfff2-2fd7-11e8-991b-339363e8489e.png)
+**2 - Saving to a Coconut File:** Clicking the save icon will download the contents of the code editor into a Coconut file called ```coconut.coco```.
 
-On the right, the user can see loading dots as the code loads in the output section:
+**3 - Code Execution:** Clicking `RUN`or using the keyboard shortcut `Ctrl + Enter` for Windows or `⌘ + Enter` for Mac will compile and execute the program.
 
-![dots](https://user-images.githubusercontent.com/35832643/37873777-0d66bf4e-2fd8-11e8-8f14-555df97d3312.png)
+**4 - Showing the Compiled Python:** Checking the box labeled `PYTHON` will show the user the compiled Python, which is the output of the Coconut compiler. the Python code will be displayed using Python syntax highlighting.
 
-On the left, we can use the code editor. The editor allows for the keyboard shortcuts from Ace editor (listed [here](https://github.com/ajaxorg/ace/wiki/Default-Keyboard-Shortcuts)). In addition, the editor includes Python and Coconut syntax highlighting for all keywords (Coconut specific keywords listed [here](http://coconut.readthedocs.io/en/latest/DOCS.html#keywords)):
+**5 - Code Editor:** The code editor, which is on the left side of the webpage, is where the user types in their Coconut code. The editor has line numbers and Coconut-specific syntax highlighting.
 
-![coco specific](https://user-images.githubusercontent.com/35832643/37873781-17561450-2fd8-11e8-8659-1c70463a48aa.png)
+**6 - Code Execution Output:** The code execution output is on the right side of the webpage. 
 
-Clicking run will compile and execute the program. The output will show up on the right. If there is a compiling error, the output will display the error from the Coconut compiler along with the traceback highlighted in red in the user's original code:
+Furthermore, if there is an error in compilation or execution, the output will display the traceback. If the error came from the Coconut compiler, the offending Coconut line will be highlighted in red in the code editor:
 
-![red tracebacks](https://user-images.githubusercontent.com/35832643/37873784-28079e7c-2fd8-11e8-8c7e-14a954f44264.png)
+![error2](https://user-images.githubusercontent.com/35832643/38785410-89d90052-40d4-11e8-93c5-8cec8fea532c.png)
 
-After the code runs, the user's code is stored in the session, so they can make changes to their existing code. 
+If the error occurred at runtime, then the offending line will be highlighted in red in the Python code:
```

Comment:
If the error occurred at runtime, then the offending line will be highlighted in red in the Python code, **if the Python checkbox is checked**.